qa/suites/fs/multiclient/tasks/mdtest.yaml

```diff
@@ -2,19 +2,33 @@
 tasks:
 - pexec:
     clients:
+      - set -x
       - cd $TESTDIR
-      - wget http://download.ceph.com/qa/mdtest-1.9.3.tgz
-      - mkdir mdtest-1.9.3
-      - cd mdtest-1.9.3
-      - tar xvfz $TESTDIR/mdtest-1.9.3.tgz
-      - rm $TESTDIR/mdtest-1.9.3.tgz
-      - MPI_CC=mpicc make
+      - sudo apt purge -y mpich
+      - sudo apt install -y mpich
```

Comment:
so, this is same as the ior pr :)

Reply:
Because because mpich package is being used here too and also we are now building ior project.